### PR TITLE
Create messages_zh_CN.properties to translate into Chinese (simplified)

### PR DESCRIPTION
**Pull request by node**
Please help to review  @oleg-nenashev  @LinuxSuRen .

### CODE REVIEW

**Comment by LinuxSuRen** [comment]
@node Perhaps the file 'messages_zh_CN.properties' should be 'message_zh_CN.properties'

File: core/src/main/resources/jenkins/CLI/WarnWhenEnabled/message_zh_CN.properties [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+<pre>
```
Comment (oleg-nenashev) [inline core/src/main/resources/jenkins/CLI/WarnWhenEnabled/message_zh_CN.properties]: hm? Does the property file support it?

File: core/src/main/resources/jenkins/CLI/WarnWhenEnabled/message_zh_CN.properties [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# The MIT License
+#
+# Copyright 2017 CloudBees, Inc.
```
Comment (oleg-nenashev) [inline core/src/main/resources/jenkins/CLI/WarnWhenEnabled/message_zh_CN.properties]: AFAIK you do not work for CloudBees (please correct me if I am wrong) :)
Please use your personal/company copyright in such case

**Review by oleg-nenashev — CHANGES_REQUESTED** [review]
It seems the copyright needs a change

File: core/src/main/resources/jenkins/CLI/WarnWhenEnabled/message_zh_CN.properties (hunk 2 above)
Comment (node) [reply core/src/main/resources/jenkins/CLI/WarnWhenEnabled/message_zh_CN.properties]: Got it.

File: core/src/main/resources/jenkins/CLI/WarnWhenEnabled/message_zh_CN.properties (hunk 1 above)
Comment (node) [reply core/src/main/resources/jenkins/CLI/WarnWhenEnabled/message_zh_CN.properties]: Updated it.

**Comment by node** [comment]
Fixed issue by reviewer's advice.

**Review by oleg-nenashev — APPROVED** [review]

**Comment by oleg-nenashev** [comment]
Thanks @node !

**Pull request merged**